pkg/kubelet/cpumanager/cpu_manager.go

```diff
@@ -58,23 +57,30 @@ type Manager interface {
 	State() state.Reader
 }
 
-func NewManager(p Policy, cr internalapi.RuntimeService, lister podLister, statusProvider status.PodStatusProvider) (Manager, error) {
-	cpuInfoFile, err := ioutil.ReadFile("/proc/cpuinfo")
-	if err != nil {
-		return nil, err
-	}
-	topo, err := discoverTopology(cpuInfoFile)
-	if err != nil {
-		return nil, err
+func NewManager(policyTYpe string, cr internalapi.RuntimeService, kletGetter kletGetter, statusProvider status.PodStatusProvider) (Manager, error) {
+	var newPolicy Policy
+
+	switch policyTYpe {
+	case "noop":
```

Comment:
lets create `type PolicyName string` and create a const in each policy file for these names rather than using raw strings everywhere